pkg/webui/console/containers/devices-table/index.js

```diff
@@ -53,10 +57,16 @@ import {
   selectDevicesTotalCount,
   selectDevicesFetching,
   selectDeviceDerivedLastSeen,
+  isOtherClusterDevice,
 } from '@console/store/selectors/devices'
 
 import style from './devices-table.styl'
 
+const m = defineMessages({
+  otherClusterTooltip:
+    'This end device is registered on a different cluster or host (`{host}`). It cannot be accessed using this Console. To access this device, use the Console of the cluster that this device was registered on.',
```

Comment:
```suggestion
    'This end device is registered on a different cluster or host (`{host}`). It cannot be accessed using this Console. To access this device, use the Console of the cluster that this end device was registered on.',
```

Sorry, overlooked this one.